Doc/library/string.rst

```diff
@@ -325,11 +325,11 @@ The general form of a *standard format specifier* is:
    align: "<" | ">" | "=" | "^"
    sign: "+" | "-" | " "
    width_and_precision: [`width_with_grouping`][`precision_with_grouping`]
-   width_with_grouping: [`width`][`grouping_option`]
-   precision_with_grouping: "." [`precision`]`grouping_option`
+   width_with_grouping: [`width`][`grouping`]
```

Comment:
Can we keep this one? The docs say "The *grouping* option". Or maybe better: `digit_group_separator`?

Reply:
`digit_group_separator` is too wordy for BNF definition. I removed `option` as other settings can be also referenced as options (for example, `sign` is referenced as such), so it's either all of them should have such postfix or none.

Reply:
Well, I think it's fine to have a wordy BNF def but it's a matter of personal taste. However, the fact that the others don't have a postfix is convincing. Ok for keeping this.